lib/API/JSON/v1/Forms.php

```diff
@@ -131,6 +131,7 @@ public function save($data = []) {
   public function previewEditor($data = []) {
     // html
     $html = FormRenderer::renderHTML($data);
+    $html = str_replace('type="submit"', 'type="submit" disabled', $html);
```

Comment:
I think this will change the appearance of the submit button in the preview. Could we use a different way of preventing submit? Maybe add `onsubmit` with `event.preventDefault` on form element.